front/cypress/e2e/admin/pages_and_menu/create_update_and_view_custom_page.cy.ts

```diff
@@ -0,0 +1,154 @@
+import { randomString } from '../../../support/commands';
+
+describe('Admin: create, update, and view custom page', () => {
+  before(() => {
+    cy.setAdminLoginCookie();
+    cy.visit('/admin/pages-menu/');
+    cy.acceptCookies();
+  });
+
+  it('creates a custom page successfully', () => {
+    cy.intercept('POST', '**/static_pages/**').as('createCustomPage');
+    cy.intercept('PATCH', '**/static_pages/**').as('updateCustomPage');
+    cy.intercept('GET', '**/static_pages/**').as('getCustomPage');
+
+    const pageName = randomString();
```

Comment:
Perhaps these can be taken out of the test itself? I think we put them in the before block normally so that they can be reused across the test file. You only have one test here so it doesn't really make a difference but perhaps could be useful if you decide to add another one.